src/dsa/rpo_falcon512/mod.rs

```diff
@@ -42,16 +39,64 @@ pub const PK_LEN: usize = 897;
 pub const SK_LEN: usize = 1281;
 
 /// Signature length as a u8 vector.
-const SIG_LEN: usize = 626;
+const SIG_LEN: usize = 625;
```

Comment:
Is `SIG_LEN` still an appropriate name for this? Maybe it should be something like `SIG_POLY_LEN`?

Reply:
Updated